src/lib/shopping-fpti/event-handlers.js

```diff
@@ -0,0 +1,104 @@
+/* @flow */
+import type { Config } from '../../types';
+import type { EventType } from '../../types/shopping-events';
+
+export function eventSifnoBuilderInit(config : Config) : Object {
```

Comment:
Can you fix this to be `eventSinfoBuilderInit` -- looks like `fn` `should be `nf`. Not a big deal but might avoid some confusion in the future.

Reply:
I'll fix it before merge